setup.py

```diff
@@ -51,7 +50,5 @@
     license='Apache',
     keywords='ITK InsightToolkit Math-morphology Label-images',
     url=r'https://github.com/InsightSoftwareConsortium/ITKLabelErodeDilate',
-    install_requires=[
-        r'itk>=5.3rc4'
-    ]
-    )
+    install_requires=[r'itk~=5.3.0']
```

Comment:
What does `~=` mean here? Why not keep `>=`?

Reply:
https://stackoverflow.com/questions/39590187/in-requirements-txt-what-does-tilde-equals-mean/39590286#39590286